conf/fluo-deploy.props.example

```diff
@@ -0,0 +1,66 @@
+# Copyright 2014 Fluo authors (see AUTHORS)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+# Cluster user name (install command will SSH to cluster using this user)
+# Leave default below if launching cluster in AWS
+cluster.username = ec2-user
+# Cluster base directory where install/ & tarballs/ directories are created
+# Leave default below if launching cluster in AWS
+cluster.base.dir = /home/ec2-user
+# Hostname of node that must direct installation of cluster.  Will be given Public IP if launching in EC2.
+# If not launching in EC2, node must have Public IP that can be reached from your machine.
+# Hostname can be chosen from "nodes" section below.
+leader.hostname = leader
+# Specifies port to use for dynamic forwarding when connecting to leader
+#leader.forwarding.port = 38585
+# Specifies if SSH, Hosts, and .bashrc should be configured on cluster
+configure.cluster = true
+# Apache mirror to download Hadoop, Zookeeper, Accumulo
+apache.mirror = http://www.gtlib.gatech.edu/pub/apache
+# Accumulo Instance name
+accumulo.instance = instance16
+# Accumluo Password
+accumulo.password = secret
+# Software versions
+fluo.version = 1.0.0-beta-1-SNAPSHOT
+hadoop.version = 2.6.0
+zookeeper.version = 3.4.6
+accumulo.version = 1.6.1
+
+[ec2]
+# EC2 region to connect to
+region = us-east-1
+# Type of AWS instance to launch by default
+default.instance.type = m3.large
+# VPC to launch instances in 
+#vpc.id = vpc-xxxxx
+# VPC Subnet to launch instances in
+#subnet.id = subnet-xxxxxx
+# Key name used to start instances
```

Comment:
Would be nice to comment that this is the name of a ssh public key uploaded to amazon.